rocketchat_API/rocketchat.py

```diff
@@ -181,12 +181,17 @@ def users_register(self, email, name, password, username, **kwargs):
     def users_get_avatar(self, user_id=None, username=None, **kwargs):
         """Gets the URL for a user’s avatar."""
         if user_id:
-            return self.__call_api_get('users.getAvatar', userId=user_id, kwargs=kwargs)
+            response = self.__call_api_get('users.getAvatar', userId=user_id, kwargs=kwargs)
         elif username:
-            return self.__call_api_get('users.getAvatar', username=username, kwargs=kwargs)
+            response = self.__call_api_get('users.getAvatar', username=username, kwargs=kwargs)
         else:
             raise RocketMissingParamException('userID or username required')
 
+        if response.status_code == 403:
```

Comment:
I would add some comment pointing to the issue as it's not straight forward the intention here and may cause confusion for others reading the code.
Code looks good to me overall :)

Reply:
added a comment.